pyramid/url.py

```diff
@@ -2,7 +2,11 @@
 
 import os
 
-from repoze.lru import lru_cache
+try:
```

Comment:
Move this to `compat.py` and import from `compat.py` instead of repeating this try/importerror everywhere.